config/mgc-install-guide/gatewayclass.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: GatewayClass
+metadata:
+  name: kuadrant-multi-cluster-gateway-instance-per-cluster
+spec:
+  controllerName: "kuadrant.io/mgc-gw-controller"
```

Comment:
I think this should either just be added as part of the manager installation (It is expected that Gateway resources exist for it anyway) or just a separate step after mgc installation. Since it;s pretty much a requirement of MGC that this exists for anything to work currently I'd probably add it to the manager install.

@eoinfennessy It's fine for now